sbt-plugin/src/main/scala/akka/grpc/sbt/GeneratorBridge.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.sbt
+
+import akka.grpc.gen.Logger
+import akka.grpc.gen.BuildInfo
+import akka.grpc.gen.CodeGenerator.ScalaBinaryVersion
+import sbt.CrossVersion
+
+object GeneratorBridge {
+  def sandboxedGenerator(
+      codeGenerator: akka.grpc.gen.CodeGenerator,
+      scalaBinaryVersion: ScalaBinaryVersion,
+      logger: Logger): protocbridge.Generator = {
+    // This matches the sbt binary version (2.12)
+    val codegenScalaBinaryVersion = CrossVersion.binaryScalaVersion(BuildInfo.scalaVersion)
+    protocbridge.SandboxedJvmGenerator(
+      codeGenerator.name,
+      protocbridge.Artifact(BuildInfo.organization, s"${BuildInfo.name}_$codegenScalaBinaryVersion", BuildInfo.version),
+      codeGenerator.suggestedDependencies(scalaBinaryVersion),
+      new SandboxedProtocBridgeSbtPluginCodeGenerator(_, codeGenerator.getClass.getName, logger))
+  }
+  def jvmGenerator(
+      codeGenerator: akka.grpc.gen.CodeGenerator,
+      scalaBinaryVersion: ScalaBinaryVersion,
+      logger: akka.grpc.gen.Logger): protocbridge.Generator = {
+    val adapter = new PlainProtocBridgeSbtPluginCodeGenerator(codeGenerator, scalaBinaryVersion, logger)
+    protocbridge.JvmGenerator(codeGenerator.name, adapter)
+  }
+
+  /** INTERNAL API */
+  class PlainProtocBridgeSbtPluginCodeGenerator(
```

Comment:
could these implementations be completely hidden?
```suggestion
  private class PlainProtocBridgeSbtPluginCodeGenerator(
```